### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by staticfloat**
@jlbuild v0.6.0-rc3

### CODE REVIEW

**Comment by jlbuild** [comment]
## Status of https://github.com/JuliaLang/julia/commit/ad290e93e4d10831c1be42cc706babaaeb49c19c builds:

| Builder Name | Build | Download |
| :----------- | :---: | :------: |
| linux32 | [COMPLETE](https://buildog.julialang.org/#/builders/41/builds/45) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x86/0.6/julia-ad290e93e4-linux32.tar.gz) | N/A |
| linux64 | [COMPLETE](https://buildog.julialang.org/#/builders/38/builds/47) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x64/0.6/julia-ad290e93e4-linux64.tar.gz) | N/A |
| linuxaarch64 | [COMPLETE](https://buildog.julialang.org/#/builders/69/builds/53) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/aarch64/0.6/julia-ad290e93e4-linuxaarch64.tar.gz) | N/A |
| linuxarmv7l | [COMPLETE](https://buildog.julialang.org/#/builders/4/builds/53) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/armv7l/0.6/julia-ad290e93e4-linuxarmv7l.tar.gz) | N/A |
| linuxppc64le | [PENDING](https://buildog.julialang.org/#/buildrequests/658) | N/A | N/A |
| osx64 | [COMPLETE](https://buildog.julialang.org/#/builders/1/builds/46) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/mac/x64/0.6/julia-ad290e93e4-mac64.dmg) | N/A |
| win32 | [PENDING](https://buildog.julialang.org/#/buildrequests/664) | N/A | N/A |
| win64 | [PENDING](https://buildog.julialang.org/#/buildrequests/662) | N/A | N/A |

**Comment by jlbuild** [comment]
## Status of https://github.com/JuliaLang/julia/commit/ad290e93e4d10831c1be42cc706babaaeb49c19c builds:

| Builder Name | Build | Download |
| :----------- | :---: | :------: |
| linux32 | [COMPLETE](https://buildog.julialang.org/#/builders/41/builds/46) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x86/0.6/julia-ad290e93e4-linux32.tar.gz) | N/A |
| linux64 | [COMPLETE](https://buildog.julialang.org/#/builders/38/builds/48) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x64/0.6/julia-ad290e93e4-linux64.tar.gz) | N/A |
| linuxaarch64 | [COMPLETE](https://buildog.julialang.org/#/builders/69/builds/54) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/aarch64/0.6/julia-ad290e93e4-linuxaarch64.tar.gz) | N/A |
| linuxarmv7l | [COMPLETE](https://buildog.julialang.org/#/builders/4/builds/54) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/armv7l/0.6/julia-ad290e93e4-linuxarmv7l.tar.gz) | N/A |
| linuxppc64le | [PENDING](https://buildog.julialang.org/#/buildrequests/666) | N/A | N/A |
| osx64 | [COMPLETE](https://buildog.julialang.org/#/builders/1/builds/47) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/mac/x64/0.6/julia-ad290e93e4-mac64.dmg) | N/A |
| win32 | [PENDING](https://buildog.julialang.org/#/buildrequests/672) | N/A | N/A |
| win64 | [PENDING](https://buildog.julialang.org/#/buildrequests/670) | N/A | N/A |

**Comment by jlbuild** [comment]
## Status of https://github.com/JuliaLang/julia/commit/ad290e93e4d10831c1be42cc706babaaeb49c19c builds:

| Builder Name | Build | Download |
| :----------- | :---: | :------: |
| linux32 | [COMPLETE](https://buildog.julialang.org/#/builders/41/builds/47) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x86/0.6/julia-ad290e93e4-linux32.tar.gz) | N/A |
| linux64 | [COMPLETE](https://buildog.julialang.org/#/builders/38/builds/49) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/x64/0.6/julia-ad290e93e4-linux64.tar.gz) | N/A |
| linuxaarch64 | N/A | N/A | N/A |
| linuxarmv7l | [COMPLETE](https://buildog.julialang.org/#/builders/4/builds/55) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/linux/armv7l/0.6/julia-ad290e93e4-linuxarmv7l.tar.gz) | N/A |
| linuxppc64le | [PENDING](https://buildog.julialang.org/#/buildrequests/674) | N/A | N/A |
| osx64 | [COMPLETE](https://buildog.julialang.org/#/builders/1/builds/48) | [Download](https://julialangnightlies.s3.amazonaws.com/test/bin/mac/x64/0.6/julia-ad290e93e4-mac64.dmg) | N/A |
| win32 | [PENDING](https://buildog.julialang.org/#/buildrequests/679) | N/A | N/A |
| win64 | [PENDING](https://buildog.julialang.org/#/buildrequests/677) | N/A | N/A |

File: README.md [hunk 1]
```diff
@@ -23,3 +23,5 @@ Tags are used to alter the default behavior of `jlbuild` somewhat.  As of this w
 * `!flags=x,y,z` will add extra flags to the `make` invocation that builds julia.  Example: `@jlbuild !filter=arm !flags=BUILD_CUSTOM_LIBCXX=1,BUILD_LLVM_CLANG=1`.
 
 Finally, Julia code can be included to be run using the newly-built version of Julia.  Binary artifacts from the build will also be posted for easy access.
+
+blah
```
Comment (staticfloat) [inline README.md:27]: @jlbuild v0.6.0-rc2

**Pull request closed without merging**